pkg/workload/workload.go

```diff
@@ -577,6 +585,12 @@ func IsEvictedByPodsReadyTimeout(w *kueue.Workload) (*metav1.Condition, bool) {
 	return cond, true
 }
 
+// IsUnsetRequeuedByClusterQueueStopped returns true if the workload is unset requeued by cluster queue stopped.
+func IsUnsetRequeuedByClusterQueueStopped(w *kueue.Workload) bool {
```

Comment:
```suggestion
func IsDisabledRequeuedByClusterQueueStopped(w *kueue.Workload) bool {
```

I mentioned above, could we re-title this function name to smoothing like `disable`?

Reply:
Renamed